rebasehelper/spec_hooks/pypi_url_fix.py

```diff
@@ -26,13 +26,16 @@
 
 
 class PyPIURLFixHook(BaseSpecHook):
-    """
-    SpecHook for transforming old nonfunctional python package url to
-    a safe url(eg. https://pypi.* to https://files.pythonhosted.*)
+    """SpecHook for transforming an old python package url to a new url(eg. https://pypi.python.org/packages/* to
+    https://files.pythonhosted.org/packages/* or https://pypi.python.org/pypi/* to https://pypi.org/project/*)
+
```

Comment:
This is fine, but the lines are too long. The 120 characters per-line limit doesn't apply to docstrings, because they can be used in different environments that may be expecting maximum of 80 characters per line, such as interactive python shell or sphinx documentation.

Reply:
Oh, thanks for pointing that out, didn't know that.